WorkingInR/0_Preamble.Rmd

```diff
@@ -0,0 +1,74 @@
+---
+title: "Preamble"
+output:
+  bookdown::html_document2:
+  highlight: tango
+toc: true
+toc_float: true
+css: ../css/style-chapters.css
+---
+  
+``` {r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE)
+```
+
+
+# Preamble
+
+## About this website
+
+These pages are the result of a collaborative effort between bioCEED, Centre for Excellence in Biology Education, the Department of Biological Sciences at the University of Bergen, and the Department of Biology and the Department of Mathematical Sciences at the Norwegian University of Science and Technology.
+
+## Who is it for?
+
```

Comment:
Mainly biology students at bachelor and master level. We also have more advanced stuff for higher levels.
Some of the stuff links directly up to BIO courses at Uib and NTNU.

Reply:
Yes. suggestion:

These pages are essentially meant for biology students both at the bachelor and master level, and link up to courses held at UiB and NTNU.
Contents may certainly be useful for other educational programmes (biomedicine, etc) and other institutions. 
Some of our pages offer more advanced content which may be relevant at doctoral levels and in research generally.